modules/features/manatime_feature_projects/manatime_feature_projects.module

```diff
@@ -43,6 +44,44 @@ function manatime_feature_projects_action_info() {
 }
 
 /**
+ * Implements hook_field_widget_form_alter().
+ */
+function manatime_feature_projects_field_widget_form_alter(&$element, &$form_state, $context) {
+  $query = drupal_get_query_parameters();
+  // Query string param.
+  if (!empty($query['id_project'])) {
+    unset($element['actions']['ief_add']);
+    unset($element['actions']['ief_add_existing']);
+    if (isset($element['#field_name']) && 'field_project' == $element['#field_name']) {
+      if (!isset($element['entities'][0])) {
+        $id_project = $query['id_project'];
+        foreach ($form_state['inline_entity_form'] as $key_state => $form_state_item) {
+          if ('projects' == $form_state_item['settings']['bundles'][0]) {
+            break;
+          }
+        }
+        $project = node_load($id_project);
```

Comment:
as suggestion, check the node bundle type to be sure you are loading a project node.